sdks/java/core/src/test/java/org/apache/beam/SdkCoreApiSurfaceTest.java

```diff
@@ -37,6 +37,7 @@ public void testSdkApiSurface() throws Exception {
     @SuppressWarnings("unchecked")
     final Set<String> allowed =
         ImmutableSet.of(
+            "com.google.common",
```

Comment:
<!--new_thread; commit:6d71ea7ce4ecf97a6e2e6564354ddc66b3817202; resolved:0-->
We shouldn't need this anymore.

Reply:
Done